styles/globals.css

```diff
@@ -28,6 +33,42 @@
     border-top-color: black;
 }
 
+.oswald{ 
+    font-family: 'Oswald';
+}
+
+.work-sans{ 
+    font-family: 'Work Sans';
+}
+
+.raleway{
+    font-family: 'Raleway';
+}
+
+.montserrat{
+    font-family: 'Montserrat';
+}
+
+.top-right {
```

Comment:
don't do positioning styles like these in globals.css, do them in the .tsx file you use them